vendor/src/github.com/guregu/null

```diff
@@ -0,0 +1 @@
+Subproject commit 41fe781af5dd0b14980914a13e7921643517d00f
```

Comment:
We're not using submodules for vendor dependencies in this project... please maintain our conventions for vendoring code until we decide to move to a different convention wholesale.